cassovary-core/src/test/scala/com/twitter/cassovary/graph/DirectedGraphSpec.scala

```diff
@@ -13,58 +13,60 @@
  */
 package com.twitter.cassovary.graph
 
-import org.specs.Specification
+import org.junit.runner.RunWith
+import org.scalatest.WordSpec
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.matchers.ShouldMatchers
 
-class DirectedGraphSpec extends Specification {
-  var graph: DirectedGraph = _
+@RunWith(classOf[JUnitRunner])
+class DirectedGraphSpec extends WordSpec with ShouldMatchers {
 
-  val twoNodeGraph = beforeContext {
-    graph = TestGraphs.g2_mutual
-  }
-
-  val sixNodeGraph = beforeContext {
-    graph = TestGraphs.g6
-  }
+  val twoNodeGraph = TestGraphs.g2_mutual
+  val sixNodeGraph = TestGraphs.g6
 
-  "two node graph with each following the other" definedAs twoNodeGraph should {
+  "two node graph with each following the other" should {
     "nodeCount and edgeCount should be correct" in {
-      graph.nodeCount mustEqual 2
-      graph.edgeCount mustEqual 2
+      twoNodeGraph.nodeCount shouldEqual 2
+      twoNodeGraph.edgeCount shouldEqual 2
     }
 
     "maxNodeId should be correct" in {
-      graph.maxNodeId mustEqual 2
+      twoNodeGraph.maxNodeId shouldEqual 2
     }
 
     "elements should return all the nodes" in {
-      graph.iterator.toList.map { _.id }.sortBy (+_) mustEqual List(1, 2)
+      twoNodeGraph.iterator.toList.map {
+        _.id
```

Comment:
I don't like new lines here. I would write it in one line as there is no serious logic in the _.id. And probably only in ( ).